local-development.md

```diff
@@ -0,0 +1,65 @@
+# Local Deployment Tesing
+
+At times it may be necessary to change the way the deployment works, the most convenient way of doing this is to make changes locally. This avoids having to setup and wait for external services. This file contains some details on how to do this development locally.
```

Comment:
A link to `elife-xpub-formula` would place this in context: this procedure helps with modifying that repository.